src/modal/modal-window.ts

```diff
@@ -1,20 +1,25 @@
 import {
+  AfterViewInit,
   Component,
-  Output,
+  ContentChild,
+  ContentChildren,
+  ElementRef,
   EventEmitter,
   Input,
-  ElementRef,
-  Renderer,
+  OnDestroy,
   OnInit,
-  AfterViewInit,
-  OnDestroy
+  Output,
+  Renderer
 } from '@angular/core';
 
 import {ModalDismissReasons} from './modal-dismiss-reasons';
 
+let nextId = 0;
+
 @Component({
   selector: 'ngb-modal-window',
   host: {
+    '[attr.aria-modal]': 'true',
```

Comment:
Here the link to the issue opened on the twbs repo: https://github.com/twbs/bootstrap/issues/22429